### DISCUSSION TITLE
如何切换到Server模型？

### DISCUSSION THREAD

**Discussion by lucasjinreal** [PaddlePaddle/PaddleOCR#13196] [Q&A]
默认模型效果较差

**Comment by GreatV** [comment]
请参考 https://github.com/PaddlePaddle/PaddleOCR/blob/main/doc/doc_ch/inference_ppocr.md#%E5%9F%BA%E4%BA%8Epython%E5%BC%95%E6%93%8E%E7%9A%84pp-ocr%E6%A8%A1%E5%9E%8B%E5%BA%93%E6%8E%A8%E7%90%86
